source/errors.rst

```diff
@@ -138,7 +138,7 @@ With multiple invalid identifiers, the error is formatted like so:
 
   {
     "status": 400,
-    "error": "Invalid identifier",
+    "error": "Invalid Identifier",
```

Comment:
https://github.com/osuosl/timesync-node/blob/develop/src/errors.js#L138 shows `'The provided identifier was invalid'` as the correct form.